packages/mco/components/discovered-application-wizard/utils/reducer.ts

```diff
@@ -34,6 +35,7 @@ export type EnrollDiscoveredApplicationState = {
     name: string;
   };
   configuration: {
+    isVolumeConsistencyEnabled?: boolean;
```

Comment:
```suggestion
    isVolumeConsistencyEnabled: boolean;
```
Is it an optional field in the CRD specification?

Reply:
Its not in the CRD specification, its optional to enable it